<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExTcpClient</class>
 <widget class="QMainWindow" name="ExTcpClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExTcpClient</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labIP">
        <property name="text">
         <string>IP address</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentText">
         <string>127.0.0.1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labPort">
        <property name="text">
         <string>Port</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="maximum">
         <number>65000</number>
        </property>
        <property name="value">
         <number>10000</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnSend">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actConnect"/>
   <addaction name="actDisconnect"/>
   <addaction name="actClear"/>
   <addaction name="actQuit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actConnect">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/Image4.png</normaloff>:/Icons/Image4.png</iconset>
   </property>
   <property name="text">
    <string>actConnect</string>
   </property>
   <property name="toolTip">
    <string>actConnect</string>
   </property>
  </action>
  <action name="actDisconnect">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/Image5.png</normaloff>:/Icons/Image5.png</iconset>
   </property>
   <property name="text">
    <string>actDisconnect</string>
   </property>
   <property name="toolTip">
    <string>actDisconnect</string>
   </property>
  </action>
  <action name="actClear">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/Image1.png</normaloff>:/Icons/Image1.png</iconset>
   </property>
   <property name="text">
    <string>actClear</string>
   </property>
   <property name="toolTip">
    <string>actClear</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/Icons/Image2.jpg</normaloff>:/Icons/Image2.jpg</iconset>
   </property>
   <property name="text">
    <string>actQuit</string>
   </property>
   <property name="toolTip">
    <string>actQuit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
